tools/tests/py2fgen/test_cli.py

```diff
@@ -83,7 +84,8 @@ def run_test_case(
 
 
 def invoke_cli(cli, module, function, library_name):
-    cli_args = [module, function, library_name]
+    rpath = f"{sys.base_prefix}/lib"
```

Comment:
I am wondering if there is a scenario where you want to use a different libpython at runtime than the one that was used when executing py2fgen. If not, we could unconditionally set the `rpath` and remove the cli parameter.

Reply:
This is what in fact seems to happen when we don't set it: My system python is 3.10. I could run 
` uv run nox -s "test_tools-3.10(unittest)" ` but `uv run nox -s "test_tools-3.11(unittest)" ` failed when not passing the rpath because it linked the system library. I think for that example that is _not_ what you want because it is undefined. But I also don't know for other scenarios.